source/learn/intrinsics/_pages/NORM2.md

```diff
@@ -47,11 +47,10 @@ Sample program:
 ```fortran
 program demo_norm2
 implicit none
-integer :: i
 
 real :: x(3,3) = reshape([ &
    1, 2, 3, &
-   4, 5, 6, &
+   4, 5 ,6, &
```

Comment:
Is this an intended change?

Reply:
No. Ships passing in the sea, I'm afraid. I incorporated #146 changes that affected the intrinsic descriptions.